noxfile.py

```diff
@@ -24,6 +24,18 @@ def _install_bundle(session: nox.Session) -> None:
         "./requirements.txt",
     )
 
+    # Installing kedro-viz to bundled/libs from here and not keeping it in requirements.in to avoid installing all its required packages
+    session.install(
+        "-t",
+        "./bundled/libs",
+        "--no-cache-dir",
+        "--implementation",
+        "py",
+        "--no-deps",
+        "--upgrade",
+        "kedro-viz",
```

Comment:
For viz, do we have a particular version that we need to pin? (I think it's the next release version, so we need to wait until it is released, but I just want to confirm this).

Reply:
Mostly it would be next release `10.0.0`. later we willl pin it